src/Entry.tsx

```diff
@@ -41,7 +40,6 @@ const Entry = ({ store }: Props) => {
         ...actionMessages,
         ...eventsMessages,
         ...systemMessages,
-        ...motionMessages,
```

Comment:
We seem to have moved away from having two separate files for actions and motions messages, which makes sense since they share the same set of messages, so have removed.